README.md

```diff
@@ -1,5 +1,7 @@
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
 
+You can [**fork it**](https://help.github.com/articles/fork-a-repo/) by using the Fork button in the top-right section of the page.
+
```

Comment:
I believe https://help.github.com/articles/viewing-branches-in-your-repository/ would be a better entry point for branch help. :wink: